banzai/images.py

```diff
@@ -88,32 +105,26 @@ def writeto(self, filename, fpack=False):
         image_hdu.header['EXTEND'] = True
         image_hdu.name = 'SCI'
         hdu_list = [image_hdu]
-        if self.catalog is not None:
-            table_hdu = fits_utils.table_to_fits(self.catalog)
-            table_hdu.name = 'CAT'
-            hdu_list.append(table_hdu)
-        if self.bpm is not None:
-            bpm_hdu = fits.ImageHDU(self.bpm.astype(np.uint8))
-            bpm_hdu.name = 'BPM'
-            hdu_list.append(bpm_hdu)
+        hdu_list = self.add_bpm_to_hdu_list(hdu_list)
+        hdu_list = self.add_data_tables_to_hdu_list(hdu_list)
```

Comment:
This order will change the structure of the output images, i.e. now HDU 1 will be the BPM and HDU 2 the catalogue. Maybe Curtis can comment on whether that matters though.

Reply:
Good catch. If that matters @cmccully , I will revert to the old order